index.html

```diff
@@ -2983,6 +2983,18 @@ <h3>Zero-Knowledge Proofs</h3>
 operations in zero-knowledge.
           </li>
         </ul>
+        <p class="note">
+When a <a>holder</a> has selectively disclosed a portion of a
+<a>verifiable credential</a>, it is important that the <a>verifier</a> compare
+the information provided in the derived <a>verifiable credential</a> and make
+sure it is compatible with the schema in the <code>credentialSchema</code>
+<a>property</a> provided by the <a>issuer</a>. It is also possible for the
+<a>verifier</a> to provide a schema to the <a>holder</a> as part of a request
+for the <a>holder</a>'s data, and for the <a>verifier</a> to ensure that the
+derived <verifiable credential</a> is compatible with that schema as well. We do
+not define such a request schema in this specification, but an example of one
+method for doing so is [[?PRES-EX]].
```

Comment:
I'm not sure I agree with this section as written. The usage of a schema does not inherently need to be tied to the securing mechanism of a credential. This means a schema could or could not be used with a selectively disclosable credential.

If the schema is not formed with selective disclosure in mind, then validation is likely to fail.

I would encourage `credentialSchema` implementations to consider the implications of SD/ZKP credentials and provide guidance for processing depending on the construction.

Reply:
> I'm not sure I agree with this section as written. The usage of a schema does not inherently need to be tied to the securing mechanism of a credential. This means a schema could or could not be used with a selectively disclosable credential.

There is nothing in what I've written that ties schema usage to a securing mechanism. Nothing in this language is normative. It is intended to give guidance where it was considered to be lacking.

> If the schema is not formed with selective disclosure in mind, then validation is likely to fail.

agreed, language added in [0573a78](https://github.com/w3c/vc-data-model/pull/1112/commits/0573a78ad393841d32000024b530255bbbc0308f)

> I would encourage `credentialSchema` implementations to consider the implications of SD/ZKP credentials and provide guidance for processing depending on the construction.

agreed, language added in [7d6a197](https://github.com/w3c/vc-data-model/pull/1112/commits/7d6a1977829a4abb78ed2be272b9b22037449ef4)

Reply:
thanks Brent, much better